src/modules/Dashboard/pages/Applet/AppletMultiInformant.tsx

```diff
@@ -37,7 +39,37 @@ export const AppletMultiInformant = () => {
     <StyledBody>
       {isLoading && <Spinner />}
       {appletData && (
-        <LinkedTabs hiddenHeader={hiddenHeader} tabs={appletTabs} isCentered={false} />
+        <>
+          <Box
+            display="flex"
+            flexDirection="row"
+            alignItems="center"
```

Comment:
A reusable component already exists that you can take advantage of: `StyledFlexTopCenter`. So you can do something like:

```tsx
<StyledFlexTopCenter
  gap={theme.spacing(1.6)}
  margin={`${theme.spacing(1.2)} ${theme.spacing(3.4)}`}>
```

Reply:
Nice, thank you!